mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-holdings-records.feature

```diff
@@ -321,29 +368,27 @@ Feature: Test quickMARC holdings records
     Then status 422
     Then match response.message == 'Is unique tag'
 
-### Uncomment scenario when resolve bug with duplicated 852 tag
```

Comment:
Which bug are you referring to?

Reply:
Bug that allow as to create multiply 852 tags [MODQM-204](https://issues.folio.org/browse/MODQM-204).
It's already resolved, so I decided to uncomment this case